addon/components/polaris-inline-error.js

```diff
@@ -0,0 +1,35 @@
+import Component from '@ember/component';
+import layout from '../templates/components/polaris-inline-error';
+import { assert } from '@ember/debug';
+
+export default Component.extend({
+  layout,
+  tagName: '',
+
+  /**
+   * Unique identifier of the invalid form field that the message describes
+   *
+   * @type {String}
+   * @default null
+   * @public
+   * @required
+   */
+  fieldID: null,
+
+  /**
+   * Content briefly explaining how to resolve the invalid form field input.
+   *
+   * @type {String|Component}
+   * @default null
+   * @public
+   */
+  message: null,
+
+  'data-test-inline-error': true,
+  'data-test-inline-error-icon': true,
+
+  init() {
+    this._super(...arguments);
+    assert('[polaris-inline-error] Missing required `fieldID` param!', this.get('fieldID'));
```

Comment:
The React component doesn't seem to care about this... why do we?

Reply:
the docs mention it's required...I'm fine with dropping this, tbh

Reply:
True... don't see anything in the code enforcing it but maybe there's something in React that requires the `props` to be present 🤷‍♂️

Reply:
TypeScript